src/server/src/main/resources/db/cassandra/024_node_metrics_v2_partitioning.cql

```diff
@@ -0,0 +1,35 @@
+--
```

Comment:
- shouldn't this file `024_node_metrics_v2_partitioning.cql` be called instead `024_node_metrics_v3_partitioning.cql`